etc/vagrant-provision.sh

```diff
@@ -20,7 +20,7 @@ apt-get install -y postgresql libpq-dev
 apt-get install -y libffi-dev
 
 # node.js
-curl -sL https://deb.nodesource.com/setup_12.x | bash -
+curl -sL https://deb.nodesource.com/setup_18.x | bash -
```

Comment:
We should also update `/.nvmrc` to specify version 18. (There'll also be updates to make in the https://github.com/demozoo/devops repo, but I'll handle that...

Reply:
noted, updated.